stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py

```diff
@@ -104,3 +104,6 @@ def test_02_stock_orderpoint_mrp_link_indirect_routing(self):
         self.assertTrue(mo2)
         self.assertEqual(mo2.orderpoint_id, self.orderpoint_secondary_loc)
         self.assertEqual(mo2.orderpoint_id, self.orderpoint_secondary_loc)
+
+    def test_03_stock_orderpoint_mrp_link_action_view(self):
+        self.orderpoint_secondary_loc.action_view_mrp_productions()
```

Comment:
The test could more extended? Maybe check that if you use the domain returned and search, the result is the expected?

Reply:
@etobella I tried something...

Reply:
Perfectly fine!